gui/gui.go

```diff
@@ -18,6 +19,19 @@ import (
 	"go.uber.org/zap"
 )
 
+/*
+#cgo CFLAGS: -x objective-c
```

Comment:
I think we can also execute this code only if system is darwin... I'm not aware how it can be done correctly in go thus... let me know I've fixed